services/scraper/src/schema/request-body-schema.ts

```
@@ -0,0 +1,22 @@
+import z, { array } from "zod";
```

ESLint

[error] 'array' is defined but never used.

(@typescript-eslint/no-unused-vars)